docs/pai-job-protocol.yaml

```diff
@@ -126,6 +133,10 @@ parameters:
   model: resnet20
   batchsize: 32
 
+secrets:
+  docker_password: password
+  api_token: cGFzc3dvcmQ=
```

Comment:
>api_token [](start = 2, length = 9)

where do you use api_token?

Reply:
it hasn't been used in this file, just shows that the secret can be either password or token.